comixed-frontend/src/assets/i18n/es/library.json

```diff
@@ -2,484 +2,485 @@
-      "show-details": "Show details.",
-      "create-reading-list": "Create a new reading list..."
+      "show-filters": "Mostrar filtros de biblioteca.",
+      "list-layout": "Mostrar como lista.",
+      "grid-layout": "Mostrar como cuadrícula.",
+      "show-navigation-tree": "Mostrar contenido de la biblioteca...",
+      "select-all": "Seleccionar todos los cómics",
+      "deselect-all": "Deseleccionar todos los cómics",
+      "scrape": "Obtener metadatos para cómics seleccionados",
+      "delete": "Eliminar cómics seleccionados",
+      "show-details": "Mostrar detalles.",
+      "create-reading-list": "Crear una nueva lista de lectura..."
     },
     "options": {
       "rows": {
-        "10-comics": "Show 10 Comics",
-        "25-comics": "Show 25 Comics",
-        "50-comics": "Show 50 Comics",
-        "100-comics": "Show 100 Comics"
+        "10-comics": "Ver 10 cómics",
+        "25-comics": "Ver 25 cómics",
+        "50-comics": "Ver 50 cómics",
+        "100-comics": "Ver 100 cómics"
       },
       "same-height": {
-        "tooltip": "Adjust the size of cover images.",
-        "label": "Use As Cover Height"
+        "tooltip": "Ajustar el tamaño de las imágenes de portado.",
+        "label": "Usar como altura de la portada"
       },
       "cover-size": {
-        "tooltip": "Set the cover size."
+        "tooltip": "Fijar la altura de la portada."
       }
     },
     "delete-comics": {
-      "header": "Delete Selected Comics",
-      "message": "Are you sure you want to delete the {count, plural, =1{selected comic} other{# selected comics}}?"
+      "header": "Eliminar los cómics seleccionados",
+      "message": "¿Seguro que quieres eliminar {count, plural, =1{el cómic seleccionado} other{los #cómics seleccionados}}?"
     }
   },
   "library-filter": {
-    "title": "Library Filters",
+    "title": "Filtros de la biblioteca",
     "option": {
-      "select-all-publishers": "All Publishers...",
-      "select-all-series": "All Series..."
+      "select-all-publishers": "Todas las Editoriales...",
+      "select-all-series": "Todas las Series..."
     },
     "label": {
-      "publisher": "Publisher",
-      "series": "Series",
-      "show-deleted": "Show Comics Marked For Deletion"
+      "publisher": "Editorial",
+      "series": "Serie",
+      "show-deleted": "Mostrar los cómics marcados para eliminación"
     },
     "button": {
-      "reset-filters": "Reset Filters"
+      "reset-filters": "Borrar filtros"
     }
   },
   "duplicate-pages-effects": {
     "get-all": {
       "error": {
-        "detail": "Failed to get all duplicate pages."
+        "detail": "Error al obtener todas las páginas duplicadas."
       }
     },
     "set-blocking": {
       "success": {
-        "detail": "Successfully marked selected pages as {blocked, select, true{blocked} other{unblocked}}."
+        "detail": "Páginas seleccionadas marcadas correctamente como {blocked, select, true{'bloqueadas'} other{'desbloqueadas'}}."
```

Comment:
Please remove the single quotes from around the words bloqueadas and desbloqueadas.

Reply:
Hopefully solved with commit [abd9ff7]